hack/shred_accounts.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+ACCOUNT_OPERATOR_NAMESPACE=aws-account-operator
+ACCOUNT_SHREDDER_NAMESPACE=aws-account-shredder
+OC_LOG_WINDOW="1h"
+
+
+function help {
+  name=`basename $0`
+  cat <<EOF
+$name [-f filename] [-w ocLogsSince] [-h] command
+
+Run an ad-hoc shredding of 1 or more AWS account ids using a locally deployed AWS Account Shredder.
+
+For more details see: https://github.com/openshift/aws-account-shredder#running-an-ad-hoc-shred
+
+Requirements:
+    1. `oc` and `osdctl` available in your $PATH
+    2. AWS Account Shredder running locally - https://github.com/openshift/aws-account-shredder#deploying-shredder-locally 
+
+commands:
+    mark        marks Account CRs of the associated AWS account ids as failed so the 
+                shredder cleans up their AWS resources.
+    status      check the shredding status of the AWS account ids.
+    cleanup     remove Account CRs of AWS account ids.
+
+options:
+    -h  print script usage message and exit
+    -f  file path to a list of AWS account ids. This file should only contain
+        AWS account ids, 1 per line.
+    -w  the "since" variable used when fetching oc logs (e.g. `oc logs --since 5m`).
+        Shorter windows will speed up runtimes. Defaults to "1h"
+EOF
+}
+
+function lowerCase {
+    echo "$1" | awk '{print tolower($0)}'
+}
+
+function awsAccountIdsFromFile {
+    local file=$1
+    while IFS=" " read -r AWS_ACC_ID
+    do
+        echo $AWS_ACC_ID
+    done < "$file"
+}
+
+function accountCrName {
+    local awsAccountId=$1
+    echo "aws-shredder-account-delete-$awsAccountId"
+}
+
+# generate a json map of AWS Account IDs to oc state which will act as a cache to reduce overall oc calls which are relatively slow
+# e.g. {"1234": "Failed", "9876": "Ready", ...}
+function ocAccountStatusCache {
+    oc get accounts -n $ACCOUNT_OPERATOR_NAMESPACE -o json | jq '.items | map({ (.spec.awsAccountID): .status.state }) | add'
+}
+
+# generate a json map of AWS Account IDs that have the "Failed to reset account status" error message in logs that 
+# indicates an account has been successfully shredded. oc logs can be quite large and this reduces the ammount of 
+# times we need jq to process the large amount of json
+# e.g. {"1234": true, "9876": true, ...}
+function accountFailedToUpdateCache {
+    oc logs deployment/aws-account-shredder -n $ACCOUNT_SHREDDER_NAMESPACE --since $OC_LOG_WINDOW | \
+    jq -R "fromjson? | . " -c | \
```

Comment:
And style wise, i've always been a fan of putting `|` at the beginning of the next line to make it easier to see that we're using a pipe:
```
oc logs deployment/aws-account-shredder -n $ACCOUNT_SHREDDER_NAMESPACE --since $OC_LOG_WINDOW \
  | jq -R "fromjson? | . " -c \
  | jq -s '[.[] | select(.msg | contains("Failed to reset account status"))] | map({(.AccountID): true}) | add'
```

Reply:
Also, make sure to group the args together: `jq -R "fromjson? | . " -c` vs `jq -cR "fromjson? | . "`